packages/documentation/src/pages/architecture/site-structure/bundles-and-code-organization.mdx

```diff
@@ -1,4 +1,41 @@
-# Bundle and Code Organization
+# Bundles and Code Organization
```

Comment:
Moved this into the architecture folder. I started editing it but decided to hold off. I added a section about the application bundling with webpack. I think it would be good to document the loaders and plugins we use and why we use them.